logging/include/Logging.hpp

```diff
@@ -67,7 +67,7 @@ class LoggerImpl {
 
     // clang-format off
     std::cout << std::put_time(std::localtime(&cur_time.tv_sec), "%FT%T.") << (int)cur_time.tv_usec / 1000
-       << "|" << LoggerImpl::LEVELS_STRINGS[l].c_str()
+       << "|" << LoggerImpl::LEVELS_STRINGS[l]
```

Comment:
Is it related to the HDR historgram?

Reply:
This is related to a compilation on MacOS with a most recent clang compiler (12.05)